### DISCUSSION TITLE
Problem with typescript and react

### DISCUSSION THREAD

**Discussion by flrnd** [tailwindlabs/tailwindcss#3728] [Kind words]
This was kind of a problem report that resulted in a PEBKAC. So, I'll take the opportunity to throw some love to everyone involved instead.
As the title says, Tailwind rocks, been a few years using it to design UI's and I really love it. Just that. Love and Peace to you, all the people creating this fantastic tool.

**Comment by simonswiss** [comment]
Awwww - thank you! 

With this title, I was worried that the category was wrong for this post - but very happy to see that it was now ❤️

Love and Peace to you as well 🙏
